client.go

```diff
@@ -418,9 +418,7 @@ func (cli *S3) ValidateBucket() error {
 			Bucket: aws.String(cli.bucketName),
 		},
 	); err != nil {
-		return NewError(fmt.Errorf("validation error for bucket: %w", err), log.Data{
-			"bucket_name": cli.bucketName,
-		})
+		return err
```

Comment:
I think the code in this function could be further optimised to not need an 'if' and just return the 'err' and thus be put back to the original before this feature branch ?